### ISSUE TITLE
can I set up this program locally?

### ISSUE THREAD

**Issue by gaoe6661** [s-macke/jor1k#176]
can I set up this program on my server? I tried to search but no result(I do not know node.js)?

**Comment by zap8600** [comment]
Yes. I've done so myself. To start the server, I ran `python3 -m http.server` in my terminal.

**Comment by pflegende** [comment]
Is the Internet tunnel working?

**Comment by zap8600** [comment]
> Is the Internet tunnel working? 

You'll have to configure that yourself, but yes, it is. You can also run that locally if I'm correct.
